src/sempy_labs/_authentication.py

```
@@ -1,26 +1,35 @@
 
     For more information on Service Principal see: `Application and service principal objects in Microsoft Entra ID <https://learn.microsoft.com/en-us/entra/identity-platform/app-objects-and-service-principals?tabs=browser#service-principal-object>`_
     """
 
+    _shorthand_scopes: Dict[str, str] = {
+        "pbi": "https://analysis.windows.net/powerbi/api/.default",
+        "storage": "https://storage.azure.com/.default",
+        "azure": "https://management.azure.com/.default",
+        "graph": "https://graph.microsoft.com/.default",
+        "asazure": "https://{region}.asazure.windows.net/.default",
+        "keyvault": "https://vault.azure.net/.default"
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)